docs/src/pages/components/data-grid/export/export.md

```diff
@@ -32,20 +49,37 @@ The following sections describes some customizations available on this interface
 <GridToolbarExport csvOptions={csvOptions} />
 ```
 
-#### Exported columns
+Each export option has its own API page:
+
+- [`csvOptions`](/api/data-grid/grid-csv-export-options/)
+- [`printOptions`](/api/data-grid/grid-print-export-options/)
+
+### Remove an export option
```

Comment:
Should we use the term `export format` ?
I feel like it would make every thing easier to read.

Right now we have `export option` here and `available export` in `Customize export selector`

Reply:
Yes, that will remove some confusion